Source/WebCore/workers/WorkerFontLoadRequest.cpp

```diff
@@ -84,10 +84,13 @@ bool WorkerFontLoadRequest::ensureCustomFontData()
             contiguousData = m_data.takeAsContiguous();
         convertWOFFToSfntIfNecessary(contiguousData);
         if (contiguousData) {
-            m_fontCustomPlatformData = FontCustomPlatformData::create(*contiguousData, m_url.fragmentIdentifier().toString());
-            m_data = WTFMove(contiguousData);
-            if (!m_fontCustomPlatformData)
+            auto fontCustomPlatformData = FontCustomPlatformData::create(*contiguousData, m_url.fragmentIdentifier().toString());
```

Comment:
Please use `RefPtr` instead of `auto`. In new code, we like to be explicit about smart pointers.